panel/links.py

```diff
@@ -490,7 +493,7 @@ def _init_callback(
         else:
             code = f"try {{ {code} }} catch(err) {{ console.log(err) }}"
 
-        src_cb = CustomJS(args=references, code=code, tags=[link_id])
+        src_cb = CustomJS(args=references, code=code, tags=[link_id])  # type: ignore [call-arg]
```

Comment:
@mattpap, errored with `error: Unexpected keyword argument "tags" for "CustomJS"  [call-arg]`, which I don't think is correct.

Reply:
Fixed in https://github.com/bokeh/bokeh/pull/14390.